src/RSS.hs

```diff
@@ -0,0 +1,95 @@
+-- |
+-- Module: RSS
+-- Author: Alexander Vershilov <alexander.vershilov@gmail.com>
+--
+--
+module RSS
+  ( Item(..)
+  , MonadRss(..)
+  , RssT
+  , runRssT
+  , initializeFeeds
+  , updateFeeds
+  , module RSS.Cache
+  ) where
+
+-- component
+import RSS.Types
+import RSS.Cache
+import Gitter.Monad
+
+-- global
+import Network.Wreq
+import Data.Foldable
+import Data.Monoid
+import qualified Data.Set as Set
+import qualified Data.Text as Text
+import qualified Data.HashMap.Strict as HM
```

Comment:
Style note: don't use excessive abbreviations in imports. `HashMap` is short enough.